### PR TITLE
Build login and signup pages

### PR DESCRIPTION
**Pull request by ishitakeshawani**
preview link: https://deploy-preview-5--memory-nomads.netlify.app/

### CODE REVIEW

File: src/pages/login-page/LoginPage.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        userData.email
+      )
+    ) {
+      setError("Please enter a valid email address");
+      return false;
+    } else {
+      setError("");
+    }
+    if (
+      userData.password === "" ||
```
Comment (anupkgurung) [inline src/pages/login-page/LoginPage.js]: suggestion you could also do !userData.password if will do the same.

**Review by anupkgurung — APPROVED** [review]
clear login, Clean code
good to merge.

File: src/contexts/auth-context.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  }, [encodedToken]);
+
+  const logOut = () => {
+    setIsLoggedIn(false);
+    localStorage.removeItem("token");
+    navigate("/");
+  };
+
+  return (
+    <authContext.Provider
```
Comment (ranitasahaa1230) [inline src/contexts/auth-context.js]: in `authContext`,  'A' should be in capital  letters

**Review by ranitasahaa1230 — APPROVED** [review]
Code looks clean
Looks good to merge

File: src/contexts/auth-context.js (hunk 2 above)
Comment (ishitakeshawani) [reply src/contexts/auth-context.js]: done

File: src/pages/login-page/LoginPage.js (hunk 1 above)
Comment (ishitakeshawani) [reply src/pages/login-page/LoginPage.js]: replaced.

**Pull request merged**